inference-engine/src/inference_engine/src/ie_network_reader.cpp

```diff
@@ -259,7 +247,7 @@ CNNNetwork details::ReadNetwork(const std::string& modelPath,
     }
     if (inputModel) {
         auto ngFunc = FE->convert(inputModel);
-        return CNNNetwork(ngFunc);
+        return CNNNetwork(ngFunc, exts);
```

Comment:
Why do we need this change?

Reply:
If we have a scenario
- add extension
- read an ONNX model
- serialize it to xml
- read the model again as xml
Passing `exts` is needed. It is tested by `CustomOpUser_ONNXImporter` unit test